### ISSUE TITLE
Filling between graphs

### ISSUE THREAD

**Issue by jmberg** [almende/vis#821]
I'm trying to implement some dynamic graph viewer that should show the hull as well as the average line, so it would look something like this (bad) example:

![image](https://cloud.githubusercontent.com/assets/600565/19643376/3d5b628e-99ea-11e6-9ca8-484c55d7f248.png) 
vis [support.sas.com](http://support.sas.com/documentation/cdl/en/graphref/63022/HTML/default/viewer.htm#gr08n11-ex.htm)

I tried fiddling with the code for a while, but it seems to require architectural changes to pass all the datasets to the draw functions (not just for bar graphs) so I gave up.

**Comment by AlexDM0** [comment]
Hi,

This is not possible at the moment no, we'll add your feature request to the list.

Regards,

Alex

**Comment by AlexDM0** [comment]
Hi, 

I've added this to the feature requests:
http://visjs.org/featureRequests.html

We cannot say when this feature will be implemented at this time. We try to collect all feature requests on our website and close the issues. This does not mean we will ignore this request. More information on this policy can be found here:

http://visjs.org/blog.html#New\ website\ for\ vis\.js\!

Regards,

Alex

**Comment by jmberg** [comment]
Thanks. Actually, it seems likely that #516 will make this far easier.

**Comment by AlexDM0** [comment]
Yes and no, Stacking assumes the space between 0 and the first graph is also filled, although styling wise this could be done with an invisible fill I suppose. I would actually argue that a band-type of graph element would be better suited for this (which would also be used with the https://github.com/almende/vis/issues/354 uncertainty areas.

Regards

**Comment by mojoaxel** [comment]
Reopening as Feature-Request issue (see #2114). 
Everybody: Please feel free to implement this!

**Comment by Lanzo** [comment]
Is it possible to print a 3 month view of a timeline calendar?
